lib/attrsets.nix

```diff
@@ -78,6 +78,9 @@ rec {
   filterAttrs = pred: set:
     listToAttrs (fold (n: ys: let v = set.${n}; in if pred n v then [(nameValuePair n v)] ++ ys else ys) [] (attrNames set));
 
+  /* Remove the named attribute from an attribute set. */
+  removeAttr = name: filterAttrs (n: x: n != name);
```

Comment:
Do you know ``builtins.removeAttrs``? With that, I think we can drop ``removeAttr``, or at least use ``-s`` in the implementation instead.